.gitignore

```diff
@@ -41,32 +41,13 @@ hs_err_pid*
 # Covers JetBrains IDEs: IntelliJ, RubyMine, PhpStorm, AppCode, PyCharm, CLion, Android Studio and WebStorm
 # Reference: https://intellij-support.jetbrains.com/hc/en-us/articles/206544839
 
-# User-specific stuff
-.idea/**/workspace.xml
-.idea/**/tasks.xml
-.idea/**/dictionaries
-.idea/**/shelf
-
-# Sensitive or high-churn files
-.idea/**/dataSources/
-.idea/**/dataSources.ids
-.idea/**/dataSources.local.xml
-.idea/**/sqlDataSources.xml
-.idea/**/dynamic.xml
-.idea/**/uiDesigner.xml
-.idea/**/dbnavigator.xml
-
-# Gradle
-.idea/**/gradle.xml
-.idea/**/libraries
+.idea/
+*.iml
```

Comment:
It is controversial to delete Idea files. After all, this project is unlikely someone will open outside Idea

Reply:
If you leave idea files, you are forcing me to configure IntelliJ the way you do as well, but that's not very friendly. I can restore the files, but usually it's not a good practice for common library to git add IDE files, from what I know it's a practice more fitting to internal team projects